### PR TITLE
Add loopflow value in cnec result

### PR DESCRIPTION
**Pull request by pengbo-ruidian**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

prepare for regression test output scope loopflow computation

### CODE REVIEW

File: data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
     private double minThresholdInA;
     private double maxThresholdInA;
 
+    private double loopflow; //loopflow value
```
Comment (murgeyseb) [inline data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java:31]: Could you also add the unit in variable name (which is a good habit I think)

File: data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java [hunk 2]
```diff
@@ -45,6 +48,7 @@ public CnecResult(double flowInMW) {
         this.maxThresholdInMW = Double.NaN;
         this.minThresholdInA = Double.NaN;
         this.maxThresholdInA = Double.NaN;
+        this.loopflow = Double.NaN;
```
Comment (murgeyseb) [inline data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java:51]: replace by:
```
this(flowInMW, Double.NaN);
```

File: data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java [hunk 3]
```diff
@@ -54,6 +58,7 @@ public CnecResult() {
...
```
Comment (murgeyseb) [inline data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java:61]: replace by:
```
this(Double.NaN, Double.NaN);
```

File: distribution/src/main/resources/config.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ hades2:
     debug: false
 
 hades2-default-parameters:
-    dcMode: true
+    dcMode: false
```
Comment (murgeyseb) [inline distribution/src/main/resources/config.yml:14]: Why did you change default configuration of iTools configuration ?

**Comment by murgeyseb** [comment]
Can you also fix the code smell ?

File: distribution/src/main/resources/config.yml (hunk 4 above)
Comment (pengbo-ruidian) [reply distribution/src/main/resources/config.yml]: reverted

File: data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java (hunk 1 above)
Comment (pengbo-ruidian) [reply data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java]: updated

File: data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java (hunk 3 above)
Comment (pengbo-ruidian) [reply data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java]: ok

File: data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java (hunk 2 above)
Comment (pengbo-ruidian) [reply data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java]: ok

**Comment by pengbo-ruidian** [comment]
> Can you also fix the code smell ?

Fixed

**Comment by pengbo-ruidian** [comment]
Moved to #245
